### ISSUE TITLE
Update to latest InertiaJS

### ISSUE THREAD

**Issue by maicol07** [tbreuss/inertia-mithril#2]
Hi,

this package is outdated. Are you planning to upgrade it?

Thanks

**Comment by tbreuss** [comment]
I'll try to do so. Have you more information what exactly has to be updated? At the time of implementing I found it difficult because there was no exact specification.

**Comment by maicol07** [comment]
> I'll try to do so. Have you more information what exactly has to be updated? At the time of implementing I found it difficult because there was no exact specification.

The adapter should be updated to use it with the latest inertia version

**Comment by tbreuss** [comment]
I see. Thanks for clarification.

**Comment by tbreuss** [comment]
When you're done with your changes https://github.com/maicol07/inertia-mithril/commits/master, would you file a pull request into this repo? This would be nice.

**Comment by maicol07** [comment]
Actually, I've republished it under my namespace (although I have to fìx some bugs) 😅

**Comment by maicol07** [comment]
@tbreuss I have some trouble at getting it working. Can you look forward to update your repo (since you are more expert than me with InertiaJS and Mithril)?
Thanks

**Comment by tbreuss** [comment]
> @tbreuss I have some trouble at getting it working. Can you look forward to update your repo (since you are more expert than me with InertiaJS and Mithril)?
> Thanks

Yes sure. I'll try to do so as soon as possible.

**Comment by maicol07** [comment]
> > @tbreuss I have some trouble at getting it working. Can you look forward to update your repo (since you are more expert than me with InertiaJS and Mithril)?
> > Thanks
> 
> Yes sure. I'll try to do so as soon as possible.
> 

👍🏻 If we work on it together maybe it will be better and faster. Check my repo, I've already made something compatible (if you want I can send a PR of the changes I've made)

**Comment by tbreuss** [comment]
A pull request would be fine. Anyway, it takes some time for me. Need a typescript crash course first ;-)

**Comment by maicol07** [comment]
I've decided to mantain my own fork of the adapter, since it has changed a lot from the original repo

**Comment by tbreuss** [comment]
Thats okay with me.

My I ask what your experience are with Mithril.js and Inertia?

**Comment by maicol07** [comment]
@tbreuss I've first discovered about Mithril using [Flarum](https://flarum.org), an open source forum software. I've developed some extensions for it and I've familiarized with Mithril mroe and more.
Inertia, instead, is a more recent one. I've found it randomly, while searching something to speed up development between Laravel and JS frontend frameworks. I'm currently using them for both personal and professional projects

@tbreuss Maybe it should be better if you link the new Mithril adapter repo in the readme and keep yours for legacy Inertia (0.1)
